test/AccountMgmt/AccountMgmtTest.php

```diff
@@ -82,6 +82,45 @@ public function testSTagListInvalidCustomerKey() {
    } // End function testShowNotFound()
 
    /**
+    * Tests create, edit, and view customer key
+    */
+   public function testCreateEditViewKey() {
```

Comment:
I'm more partial to breaking unit tests up into logical pieces (one test for create, one for edit, one for view, one+ for error conditions), but this seems to be the going convention in the existing tests...